src/aat/java/uk/gov/hmcts/reform/em/orchestrator/functional/AutomatedBundlingScenarios.java

```diff
@@ -5,8 +5,10 @@
 import io.restassured.response.Response;
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.Ignore;
```

Comment:
For now, pls remove Ignored test cases. As we have to give unnecessary justification to release team why this test been ignored.